<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerpientesEscaleras</class>
 <widget class="QMainWindow" name="SerpientesEscaleras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerpientesEscaleras</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-9</y>
      <width>581</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_turno">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tirar dado</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_dado">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
